files/es/web/html/block-level_elements/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Elementos en bloque
+slug: Web/HTML/Block-level_elements
+tags:
+  - Guía
+  - HTML
+  - HTML5
+  - Principiante
+  - Web
+  - desarrollo
+translation_of: Web/HTML/Block-level_elements
+---
+Los elementos, en HTML (lenguaje de marcas de hipertexto - **Hypertext Markup Language**) usualmente son elementos "en bloque" o [elementos "en línea"](/es/docs/Web/HTML/Elementos_en_línea). Un elemento en bloque ocupa todo el espacio de su elemento padre (contenedor), creando así un "bloque". Este artículo ayuda a explicar lo que esto significa.
+
+Los navegadores suelen mostrar el elemento a nivel de bloque con un salto de línea antes y después del elemento. El siguiente ejemplo demuestra la influencia elementos en bloque:
+
+## Ejemplo en bloque
+
+### HTML
+
+```html
+<p>Este párrafo es un elemento en bloque; este fondo se ha coloreado para mostrar elemento principal (o padre) del párrafo.</p>
+```
+
+### CSS
+
+```css
+p { background-color: #8ABB55; }
+```
+
+{{ EmbedLiveSample('Block-level_Example') }}
```

Comment:
To fix example

````suggestion
## Ejemplo en bloque

HTML

```html
<p>Este párrafo es un elemento en bloque; este fondo se ha coloreado para mostrar elemento principal (o padre) del párrafo.</p>
```

CSS

```css
p { background-color: #8ABB55; }
```

{{ EmbedLiveSample('Block-level_Example') }}
````